src/quacc/recipes/common/phonons.py

```diff
@@ -5,34 +5,36 @@
 from importlib.util import find_spec
 from typing import TYPE_CHECKING
 
+from ase.atoms import Atoms
 from monty.dev import requires
 
 from quacc import job, subflow
-from quacc.atoms.phonons import get_phonopy, phonopy_atoms_to_ase_atoms
+from quacc.atoms.phonons import (
+    get_atoms_supercell_by_phonopy,
+    get_phonopy,
+    phonopy_atoms_to_ase_atoms,
+)
 from quacc.runners.phonons import PhonopyRunner
 from quacc.schemas.phonons import summarize_phonopy
+from quacc.utils.dicts import recursive_dict_merge
 
 has_phonopy = bool(find_spec("phonopy"))
 has_seekpath = bool(find_spec("seekpath"))
 
 if TYPE_CHECKING:
     from typing import Any
 
-    from ase.atoms import Atoms
-
     from quacc import Job
     from quacc.types import PhononSchema
 
-    if has_phonopy:
-        from phonopy import Phonopy
-
 
 @subflow
 @requires(has_phonopy, "Phonopy must be installed. Run `pip install quacc[phonons]`")
 @requires(has_seekpath, "Seekpath must be installed. Run `pip install quacc[phonons]`")
 def phonon_subflow(
-    atoms: Atoms,
+    displaced_atoms: Atoms,
```

Comment:
As an aside, `displaced_atoms` is a bit awkward of a argument name here because it sounds as if the `Atoms` have already been displaced (past tense) when in reality they are _going_ to be displaced. Not sure I have better alternative per se but just commenting.